README.org

```diff
@@ -59,10 +59,23 @@ capabilities]].
 
 Installing to the homescreen on:
 
-- iOS: Open organice in Mobile Safari. Tap the "share" button and
+- **iOS:** Open organice in Mobile Safari. Tap the "share" button and
   select "Add to Home Screen".
-- Android: Open organice in Chrome or Firefox. Tap the "menu" button
-  and select "Add to homescreen".
+
+- **Android:** Open organice in Chrome or Firefox. Tap the "menu"
+  button (three vertically stacked dots) and select "Add to
+  homescreen".
+
+  However, you can only do this once.  If you want multiple organice
```

Comment:
I don't know if that's relevant doc for organice. iOS and Android move so fast. Also, Android is quite fragmented. To me, it sounds like a piece of doc that in the best case might be correct now, but could soon be outdated. In any case, I cannot keep it up to date as I'm not an Android user.

@schoettl What do you think? Iirc you use Android. If you think my impression is wrong, I'll happily defer to you!

Reply:
Android doesn't change *that* fast :stuck_out_tongue_closed_eyes:  But the current text in `master` about multiple icons actually looks somewhat inaccurate to me on both Chrome and Firefox:

- On Chrome, I'm pretty sure there is no way to add multiple icons without a third-party app; at least, there is no obvious way I can find.
- On Firefox, there are two different ways to install:
    1. The "home/plus" icon in the address bar, but this creates an app icon which launches the app, and doesn't jump directly to the page from which the icon was created.
    2. Menu (three dots) > Page > Add Page Shortcut, which *does* create an icon linking to the current page, but it's just a browser bookmark which launches the browser not the frameless PWA.

Of course, any/all of this might be different for different versions of Chrome / Firefox / Android.

However, regarding the debate of what to include, let's consider two scenarios:

- We keep the documentation minimal, for fear of the information we give being wrong or becoming out of date.  Result: users are completely on their own, and might incorrectly conclude that it's impossible to achieve one or more app icons which jump directly to specific pages.
- We give a best attempt to document the current state, highlighting that the info may be incomplete or out of date, and asking readers to let us know if so.  Result: the docs are probably helpful for some/most Android users, and in the worst case, readers still know that it *was* possible to achieve multiple/specific icons in some circumstances, and are encouraged to work with us to improve the docs where needed.  With luck the existing docs may even provide them with the hints / ideas they need to figure it out.

I prefer the second scenario, so I've force-pushed a new version which takes this approach.  Hope you like it!